### DISCUSSION TITLE
Module '"react-hook-form"' has no exported member 'UseFormMethods'.  TS2305

### DISCUSSION THREAD

**Discussion by luanecavalcantizup** [react-hook-form/react-hook-form#6153] [Q&A] [answered]
Hi, I am getting this import error `Module '"react-hook-form"' has no exported member 'UseFormMethods'.  TS2305`

I appreciate any help, thank you

package.json:
`"react-hook-form": "^7.12.2"`
`"typescript": "^4.3.5"`

**Comment by bluebill1049** [comment, marked as answer]
https://react-hook-form.com/ts#UseFormReturn

**Reply by luanecavalcantizup** [reply]
Thanks @bluebill1049 :D

**Reply by bluebill1049** [reply]
welcome.
